src/components/ha-card.ts

```diff
@@ -26,6 +26,7 @@ class HaCard extends LitElement {
         );
         color: var(--primary-text-color);
         display: block;
+        overflow: hidden;
```

Comment:
Can you, 100% sure, guarantee that `ha-card` will never be used in a context where we want things to overflow? Because that's what going to happen when we change a core component. I think that we will break some use cases, and that's bad. 

I think users of `ha-card` should set `overflow: hidden` in their CSS.

Paper-card doesn't have `overflow: hidden`. https://github.com/PolymerElements/paper-card/blob/master/paper-card.js

Reply:
Good point. I'll tweak picture-elements instead.